rdr_service/cron_default.yaml

```diff
@@ -109,6 +109,11 @@ cron:
   timezone: America/New_York
   schedule: every day 06:00
   target: offline
+- description: Genomic GEM A1 Workflow
+  url: /offline/GenomicGemA1Workflow
+  timezone: America/New_York
+  schedule: every tuesday 12:00
```

Comment:
As of 13:45 today, 1/4/2021, Color will be updating their processing time to 6 AM. This sets our A1 job runs at 12, noon. Based on this new information, we will need to update this to be earlier. I suggest 00:00. I just found out about this via Slack from Cynthia, a PM for Color.